.github/workflows/build.yml

```diff
@@ -0,0 +1,32 @@
+name: build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7] #, 3.8, 3.9]
```

Comment:
is this for speed of execution or does the code not currently run on 3.8/9?

Reply:
It was for testing.  I should put these back.